src/Our.Umbraco.Ditto/Attributes/UmbracoPropertyAttribute.cs

```diff
@@ -51,5 +52,10 @@ public UmbracoPropertyAttribute(string propertyName, string altPropertyName = ""
         /// Gets or sets the default value.
         /// </summary>
         public object DefaultValue { get; set; }
+
+        /// <summary>
+        /// Defines if property is an Archetype
+        /// </summary>
+        public bool IsArchetype { get; set; }
```

Comment:
I think this binds Ditto too much with Archetype (not in terms of assembly reference, but conceptually).

Should we consider other 3rd party property-editors with complex data? `IsNuPicker`, `IsSirTrevor`, `IsVorto`
It makes Ditto non-generic.

Let's look for an alternative solution - preferably using TypeConverters.